packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -334,6 +334,7 @@ const actions = [
     SUITE.RECEIVE_AUTH_CONFIRM,
     SUITE.UPDATE_PASSPHRASE_MODE,
     SUITE.ADD_BUTTON_REQUEST,
+    SUITE.REMEMBER_DEVICE,
```

Comment:
hmm this might not be needed actually

Reply:
it depends. 
it's needed if you want to reload `suite.selectedDevice`, so if you force-remembered and want to have this information later on (some checking if its force remembered)